image_classification/mobilenet_nchw.js

```diff
@@ -26,25 +26,24 @@ export class MobileNetV2Nchw {
       inputLayout: 'nchw',
       labelUrl: './labels/labels1000.txt',
       inputDimensions: [1, 3, 224, 224],
-      dataType: 'float32',
     };
     this.outputDimensions = [1, 1000];
   }
 
   async buildConv_(input, name, relu6 = true, options = {}) {
     let weights;
-    if (this.targetDataType_==='float32') {
+    if (this.dataType_ === 'float32') {
       weights = buildConstantByNpy(this.builder_,
-          `${this.weightsUrl_}conv_${name}_weight.npy`, this.targetDataType_);
```

Comment:
You can simplify this code by passing `this.dataType_` to `buildConstantByNpy` directly.